### PR TITLE
Improve Getting Started guide usability

### PR DESCRIPTION
**Pull request by SariSaar**
- Add mentions of FTW hourly and FTW product in the guide
- Add mention of search schemas needing an update
- Update Stripe setup visuals

### CODE REVIEW

**Comment by shareoc** [comment]
Looks good.

File: src/docs/introduction/getting-started-with-ftw-daily/index.md [hunk 1]
```diff
@@ -49,6 +54,21 @@ basic development tooling:
...
    ```
 
+   <extrainfo title="FTW-hourly and FTW-product">
+   To clone FTW-hourly, use the command
+
+   ```bash
+   git clone https://github.com/sharetribe/ftw-hourly.git
+   ```
+
+   To clone FTW-Product, use the command
```
Comment (lyyder) [inline src/docs/introduction/getting-started-with-ftw-daily/index.md:64]:
```suggestion
   To clone FTW-product, use the command
```

File: src/docs/introduction/getting-started-with-ftw-daily/index.md [hunk 2]
```diff
@@ -153,15 +188,16 @@ integrations work.
...
+up for your free Flex account at
+https://www.sharetribe.com/products/flex/
 
 When you get access, you will be able to log into Flex Console and check
 the client ID.<br /> Flex Console: _Build > Applications_
 
 In addition, FTW templates use a transaction process that includes a
 privileged transition. That makes it possible to customize pricing on
-the small server that's included to the template. **Client secret** is
+the small server that's included in the template. **Client secret** is
```
Comment (lyyder) [inline src/docs/introduction/getting-started-with-ftw-daily/index.md:200]: Should we replace _small_ with _Node_ or something? Or just leave it out? Just a thought.

File: src/docs/introduction/getting-started-with-ftw-daily/index.md [hunk 3]
```diff
@@ -258,32 +258,34 @@ to learn how to apply for Stripe Connect review.
...
 
-1. After a few seconds, you should see your Connect dashboard. From the
-   left sidebar, go to **Settings** then **Connect settings**, at
-   https://dashboard.stripe.com/settings
+3. Now when you click the gear icon on the top bar to go to **Settings**
+   at https://dashboard.stripe.com/settings, you will see a new
+   **Connect** section. Click **Settings** in that section.
 
-   ![Connect settings](./stripe-connect-settings.jpg)
+![Connect settings](./stripe-connect-settings.png)
```
Comment (lyyder) [inline src/docs/introduction/getting-started-with-ftw-daily/index.md:275]: Indentation

File: src/docs/introduction/getting-started-with-ftw-daily/index.md [hunk 4]
```diff
@@ -258,32 +258,34 @@ to learn how to apply for Stripe Connect review.
...
 
-1. Make sure that **Custom** is enabled in the **Account types** section
+3. Make sure that **Custom** is enabled in the **Account types** section
```
Comment (lyyder) [inline src/docs/introduction/getting-started-with-ftw-daily/index.md:277]:
```suggestion
4. Make sure that **Custom** is enabled in the **Account types** section
```

Or, with Markdown ordered lists, all entries can have entry number `1.`. Markdown will render the correct number and then there's no need to manually update the numbering if the entries change. Like it was in this list before.

File: src/docs/introduction/getting-started-with-ftw-daily/index.md [hunk 5]
```diff
@@ -308,7 +310,7 @@ and not **pk_live**\<somethinghere\>
 ![Add Stripe secret key to Console](./add-stripe-to-console.png)
 
 > **Note**: secret key and publishable key needs to match with each
```
Comment (lyyder) [inline src/docs/introduction/getting-started-with-ftw-daily/index.md:312]: s/needs/need ?

**Review by lyyder — COMMENTED** [review]
Looks good, nice work!

File: src/docs/introduction/getting-started-with-ftw-daily/index.md (hunk 5 above)
Comment (SariSaar) [reply src/docs/introduction/getting-started-with-ftw-daily/index.md]: Good catch, fixed!

File: src/docs/introduction/getting-started-with-ftw-daily/index.md (hunk 4 above)
Comment (SariSaar) [reply src/docs/introduction/getting-started-with-ftw-daily/index.md]: Fixed to entry 1 for all entries

File: src/docs/introduction/getting-started-with-ftw-daily/index.md (hunk 2 above)
Comment (SariSaar) [reply src/docs/introduction/getting-started-with-ftw-daily/index.md]: Replaced `small` with `Node`

File: src/docs/introduction/getting-started-with-ftw-daily/index.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Getting started
 slug: getting-started-with-ftw-daily
-updated: 2020-06-25
+updated: 2021-12-09
```
Comment (SariSaar) [inline src/docs/introduction/getting-started-with-ftw-daily/index.md:4]:
```suggestion
updated: 2021-12-14
```

**Pull request merged**